sw/device/lib/testing/hmac_testutils.h

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_HMAC_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_HMAC_TESTUTILS_H_
+
+#include <assert.h>
+#include <stdint.h>
+
+#include "sw/device/lib/dif/dif_hmac.h"
+
+// Tests need to understand the byte order of the data in the string and
```

Comment:
I hardly think this is necessary. We will never support big-endian platforms. You can just annotate things that are endian hostile as `// NOTE: endian hostile`.

It also removes leaking compiler-specific defines into random files outside of libbase.

Reply:
Removed the macro in the other PR.